.cargo/config.toml

```diff
@@ -12,6 +12,9 @@ test-all = "test --all-features --all-targets"
 # Build and run all docs tests
 test-docs = "test --all-features --doc"
 
+# Build and run docs tests with default features
+test-docs-defaults = "test --doc"
```

Comment:
question: Won't these fail to run since the features won't exist?

Reply:
This is just `cargo test --doc`, which runs docs tests with default features

Previously, we used cargo-all-features to run docs tests on all feature permutations. With this change, we are only going to test docs with all features and with default features.

Reply:
Right, but are the docs tests running with default features? I thought they need all-features to run.

Reply:
The docs tests currently run, and pass, with test-all-features, which runs all feature permutations. My change is to run tests with only two of those permutations (default features and all features).

Reply:
Ah, I see. So we expect this to fail for now without all-features.

Reply:
We expect all docs tests to pass when run with default features OR `--all-features`, but not when run with all arbitrary feature permutations, including `--no-default-features`.